src/libudev/libudev.c

```diff
@@ -173,7 +173,9 @@ _public_ struct udev *udev_new(void) {
                                 if (prio < 0)
                                         log_debug("/etc/udev/udev.conf:%u: invalid log level '%s', ignoring.", line_nr, val);
                                 else
+#ifdef UDEV_PROPAGATE_LOG
                                         log_set_max_level(prio);
+#endif
```

Comment:
This guards `log_set_max_level` of course. But why should we parse `/etc/udev/udev.conf` at all?